ee/tabby-webserver/src/service/job.rs

```diff
@@ -45,12 +46,37 @@ impl JobService for DbConn {
         let (limit, skip_id, backwards) = graphql_pagination_to_filter(after, before, first, last)?;
         let rowids = ids.map(|ids| ids.into_iter().filter_map(|x| x.as_rowid().ok()).collect());
         Ok(self
-            .list_job_runs_with_filter(rowids, limit, skip_id, backwards)
+            .list_job_runs_with_filter(rowids, jobs, limit, skip_id, backwards)
             .await?
             .into_iter()
             .map(Into::into)
             .collect())
     }
+
+    async fn compute_job_run_stats(&self, jobs: Option<Vec<String>>) -> Result<JobStats> {
+        let jobs = self
+            .list_job_runs(None, jobs, None, None, None, None)
+            .await?;
+        let mut stats = JobStats {
+            success: 0,
+            failed: 0,
+            pending: 0,
+        };
+
+        for job in jobs {
+            if job.finished_at.is_none() {
```

Comment:
```suggestion
            if job.exit_code.is_none() {
```

might consider use a match clause